lib/lndclient/LndClient.ts

```diff
@@ -290,6 +289,21 @@ class LndClient extends BaseClient {
     return this.unaryCall<lndrpc.SendToRouteRequest, lndrpc.SendResponse>('sendToRouteSync', request);
   }
 
+  /**
+   * Subscribe to invoices.
+   */
+  private subscribeInvoices = (): void => {
+    if (this.invoiceSubscription) {
+      this.invoiceSubscription.cancel();
+    }
+
+    this.invoiceSubscription = this.lightning.subscribeInvoices(new lndrpc.InvoiceSubscription(), this.meta)
+    .on('error', (error) => {
+      this.setStatus(ClientStatus.Disconnected);
+      this.logger.error(`lnd has been disconnected at ${this.uri}, error: ${error}`);
```

Comment:
We want to start attempt to reconnect after this I believe, like so `this.reconnectionTimer = setTimeout(this.verifyConnection, LndClient.RECONNECT_TIMER);`